app/views/admin/questions/_question.html.slim

```diff
@@ -8,13 +8,13 @@
         = question.topic.title
     td
       =link_to admin_question_path(question)
-        = truncate(question.content, :length => 50)
+        = truncate(question.content, :length => 70)
     td
       =link_to admin_question_path(question)
-        = truncate(question.answer, :length => 50)
+        = render :partial => "admin/questions/level", :locals => { :@question => question }
```

Comment:
When this partial rendered in all questions - everything OK, but in topics we became error. That's why i must pass variable into partial.